.gitlab-ci.yml

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2022 Nitrokey GmbH
```

Comment:
Are you going to want to keep this file and others like it, e.g. the `.reuse/dep5` below?